docs/export_kernels.rst

```diff
@@ -0,0 +1,217 @@
+Export Taichi kernels to C source
+=================================
+
+The C backend of Taichi allows you to **export Taichi kernels to C source**.
+
+The exported Taichi program, which are simply C99 compatible source,
+does not necessary to be launched from Python. Instead, you may use them in
+your C/C++ project. Or even calling them in Javascript via Empscripten.
+
+Each C function corresponds to a Taichi kernel.
+For example, ``Tk_init_c6_0()`` may corresponds to ``init()`` in ``mpm88.py``.
+
+Only the source is need when copying, all the required Taichi runtimes are
+included in that single C source file.
```

Comment:
```
Only the source is needed when copying, all the required Taichi runtimes are
included in that single C source file.
```
Btw, sorry for my bad understanding, but I feel a little hard to catch the point :(
What is `copying` here? Do we need to clarify it slightly?